src/reducers/index.js

```diff
@@ -67,17 +68,19 @@ export const populatePropertyDefaults = (propertyTemplate) => {
 }
 
 export const refreshResourceTemplate = (state, action) => {
-  const newState = { ...state }
+  const rtId = Object.keys(state.resource).pop()
+  const newResource = { resource: { [rtId]: state.resource[rtId] } }
```

Comment:
I'm not familiar with this syntax (`[rtId]`).  does that evaluate the value of `rtId`?

Reply:
Yes, I just found out about it myself. It's the "Object Literal Syntax". See https://www.javascripttutorial.net/es6/object-literal-extensions/